src/main/java/frc/robot/commands/autonomous/TwoCycleProcessor2.java

```diff
@@ -31,13 +31,19 @@ public TwoCycleProcessor2(RobotContainer robotContainer) {
               Commands.sequence(AutoBuilder.resetOdom(startingPose), AutoBuilder.followPath(E_CSP))
                   .withName("E to CSP"));
 
-      event("atCSP")
+      event("atCSP_E")
           .onTrue(
               Commands.sequence(
+                      robotContainer.prepCoralIntake(),
```

Comment:
Hard to know if this will work without testing.
Is there code that we should share between the two auton branches